packages/plugin-commands-init/src/init.ts

```diff
@@ -1,62 +1,84 @@
 import fs from 'fs'
 import path from 'path'
-import { docsUrl } from '@pnpm/cli-utils'
 import { type UniversalOptions } from '@pnpm/config'
 import { PnpmError } from '@pnpm/error'
 import { writeProjectManifest } from '@pnpm/write-project-manifest'
-import renderHelp from 'render-help'
-import { parseRawConfig } from './utils'
+import { findWorkspaceDir } from '@pnpm/find-workspace-dir'
+import { readWorkspaceManifest } from '@pnpm/workspace.read-manifest'
+import { logger } from '@pnpm/logger'
+import { getMetaOptions, parseRawConfig } from './utils'
+import type { OptionsRaw } from './options'
+export { help, rcOptionsTypes, cliOptionsTypes } from './options'
 
-export const rcOptionsTypes = cliOptionsTypes
+export const commandNames = ['init', 'innit']
 
-export function cliOptionsTypes (): Record<string, unknown> {
-  return {}
-}
-
-export const commandNames = ['init']
-
-export function help (): string {
-  return renderHelp({
-    description: 'Create a package.json file',
-    descriptionLists: [],
-    url: docsUrl('init'),
-    usages: ['pnpm init'],
-  })
-}
+const getManifestDefaults = () => ({
```

Comment:
our convention is to use regular functions
```suggestion
function getManifestDefaults () {
```